SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularityScheduledJobPoller.java

```diff
@@ -112,10 +112,20 @@ public void runActionOnPoll() {
 
       final CronExpression cronExpression;
 
+      Optional<String> maybeSchedule = request.getRequest().getQuartzScheduleOrSchedule();
+      if (!maybeSchedule.isPresent()) {
```

Comment:
It is. This PR was prompted by something that was missing fro the validator causing errors later on. I guess the question is if we want to let those errors happen and fix the validator, or catch them like this.

@Calvinp still not sure this refactor is worth it to just get the 'safe' in the method name correct. Maybe let's come back to this one another time?